ctapipe/image/muon/processor.py

```diff
@@ -56,7 +62,14 @@ class RingQuery(QualityQuery):
             ("min_pixels", "np.count_nonzero(mask) > 50"),
             ("ring_containment", "parameters.containment > 0.5"),
         ],
-        help=QualityQuery.quality_criteria.help,
+        help=(
+            "list of tuples of ('<description', 'expression string') to accept "
```

Comment:
Same as above, mention explicitly that this is used to determine which events are fed into the intensity fit.

Reply:
As in:

```
Quality cuts given as list of tuples of ('description', 'expression string'), e.g. ``[('mycut', 'x > 3'),]``, 
to select fitted muons for further intensity fitting. 
You may use ``numpy`` as ``np`` and ``astropy.units`` as ``u``, but no other modules. 
DL1 image parameters can be accessed by prefixing the wanted parameter with ``dl1_params``.
```

?

Reply:
The dl1_params are not accessed by the RingQuery, are they?

```python
checks = self.ring_query(parameters=parameters, ring=ring, mask=mask)
```

Reply:
Ofc, Copy/Past mistake :sweat_smile: 

I've updateded the proposed wording.